app/views/cases/_new_form_case_type.html.slim

```diff
@@ -0,0 +1,7 @@
+/ - if params[:case][:type] == 'Case::FOI'
+= f.radio_button_fieldset :type,
+            choices: [Case::FOI,
+                      Case::FOI::TimelinessReview,
+                      Case::FOI::ComplianceReview]
+/ - else
+/   = f.hidden_field :type
```

Comment:
Do we still need the commented-out bits here?

Reply:
I think we need them in but not commented out, it's so we don't ask about internal reviews on when we make a SAR

Reply:
I'm going to leave this as-is and sort this out as part of the next ticket, CT-1391.